src/universal/styles/ui.js

```diff
@@ -7,12 +7,14 @@ const ui = {
   actionCardBgColor: theme.palette.light60l,
   cardBorderColor: theme.palette.mid30l,
   cardBorderRadius: '.25rem',
-  cardPaddingBase: '.5rem',
   cardMinHeight: '7.5rem',
+  cardPaddingBase: '.5rem',
```

Comment:
I think these are all obvious enough. As you add them, you may want to comment some of them for future posterity/maintainability.